src/app/confirmation/[confirmationKey]/page.js

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { useEffect, useState } from 'react';
+import { useRouter, useParams } from "next/navigation";
+import { useMutation } from "react-query";
+import { Error } from "@/app/activities/[activityId]/edit/components/formikFields";
+import { Container, CircularProgress } from '@mui/material';
+import { emailConfirmation } from "@/app/api.mjs";
+
+export default function EmailConfirmation({params}) {
```

Comment:
Why not just ?
`export default function EmailConfirmation({confirmationKey}) { `

Reply:
It is not working. confirmationKey is undefined